composer.json

```diff
@@ -16,19 +16,27 @@
         "php": "^8.1",
         "phpcr/phpcr-migrations": "^1.1",
         "phpcr/phpcr-implementation": "^2.1",
-        "doctrine/phpcr-bundle": "^1.3 || ^2.0",
-        "symfony/config": "^5.4 || ^6.0 || ^7.0",
-        "symfony/console": "^5.4 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
-        "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0"
+        "doctrine/phpcr-bundle": "^1.3 || ^2.0 || ^3.0",
+        "symfony/config": "^6.4 || ^7.0",
```

Comment:
with symfony < 6.4, we end up with a too old version of the symfony-cmf/testing bundle that is not compatible with phpcr-odm 2. is this an acceptable solution for you?

if not i would probably need to backport phpcr-odm 2 support into testing version 4...

Reply:
I did have discussion with @chirimoya lately and we did say that we need still support Symfony 5.4 as its a LTS is still running.

Reply:
ok, i fixed symfony-cmf/testing 4 to work with symfony 5.4 to 7. it was a bit of a mess to figure it out but in the end did not need many hacks.